frontend/src/data/nsf23313_2_1/dcids/pct.ts

```diff
@@ -0,0 +1,96 @@
+    'Percent_Female_Citizen_HispanicOrLatino_In_Count_Student_ScienceOrEngineeringOrHealth_MastersDegreeOrDoctorateDegree',
+    'Percent_Female_Citizen_HispanicOrLatino_In_Count_Student_ScienceOrEngineeringOrHealth_MastersDegree',
+    'Percent_Female_Citizen_HispanicOrLatino_In_Count_Student_ScienceOrEngineeringOrHealth_DoctorateDegree',
+    'Percent_Female_Citizen_NotHispanicOrLatino_AmericanIndianOrAlaskaNativeAlone_In_Count_Student_ScienceOrEngineeringOrHealth_MastersDegreeOrDoctorateDegree',
+    'Percent_Female_Citizen_NotHispanicOrLatino_AmericanIndianOrAlaskaNativeAlone_In_Count_Student_ScienceOrEngineeringOrHealth_MastersDegree',
+    'Percent_Female_Citizen_NotHispanicOrLatino_AmericanIndianOrAlaskaNativeAlone_In_Count_Student_ScienceOrEngineeringOrHealth_DoctorateDegree',
+    'Percent_Female_Citizen_NotHispanicOrLatino_Asian_In_Count_Student_ScienceOrEngineeringOrHealth_MastersDegreeOrDoctorateDegree',
+    'Percent_Female_Citizen_NotHispanicOrLatino_Asian_In_Count_Student_ScienceOrEngineeringOrHealth_MastersDegree',
+    'Percent_Female_Citizen_NotHispanicOrLatino_Asian_In_Count_Student_ScienceOrEngineeringOrHealth_DoctorateDegree',
+    'Percent_Female_Citizen_NotHispanicOrLatino_Black_In_Count_Student_ScienceOrEngineeringOrHealth_MastersDegreeOrDoctorateDegree',
+    'Percent_Female_Citizen_NotHispanicOrLatino_Black_In_Count_Student_ScienceOrEngineeringOrHealth_MastersDegree',
+    'Percent_Female_Citizen_NotHispanicOrLatino_Black_In_Count_Student_ScienceOrEngineeringOrHealth_DoctorateDegree',
+    'Percent_Female_Citizen_NotHispanicOrLatino_NativeHawaiianOrOtherPacificIslanderAlone_In_Count_Student_ScienceOrEngineeringOrHealth_MastersDegreeOrDoctorateDegree',
+    'Percent_Female_Citizen_NotHispanicOrLatino_NativeHawaiianOrOtherPacificIslanderAlone_In_Count_Student_ScienceOrEngineeringOrHealth_MastersDegree',
+    'Percent_Female_Citizen_NotHispanicOrLatino_WhiteAlone_In_Count_Student_ScienceOrEngineeringOrHealth_MastersDegreeOrDoctorateDegree',
+    'Percent_Female_Citizen_NotHispanicOrLatino_WhiteAlone_In_Count_Student_ScienceOrEngineeringOrHealth_MastersDegree',
+    'Percent_Female_Citizen_NotHispanicOrLatino_WhiteAlone_In_Count_Student_ScienceOrEngineeringOrHealth_DoctorateDegree',
+    'Percent_Female_Citizen_NotHispanicOrLatino_TwoOrMoreRaces_In_Count_Student_ScienceOrEngineeringOrHealth_MastersDegreeOrDoctorateDegree',
+    'Percent_Female_Citizen_NotHispanicOrLatino_TwoOrMoreRaces_In_Count_Student_ScienceOrEngineeringOrHealth_MastersDegree',
+    'Percent_Female_Citizen_NotHispanicOrLatino_TwoOrMoreRaces_In_Count_Student_ScienceOrEngineeringOrHealth_DoctorateDegree',
+    'Percent_Female_Citizen_UnknownEthnicity_UnknownRace_In_Count_Student_ScienceOrEngineeringOrHealth_MastersDegreeOrDoctorateDegree',
+    // Typo in dcid 'UnknwonRace', commented until fixed 'Percent_Female_Citizen_UnknownEthnicity_UnknwonRace_In_Count_Student_ScienceOrEngineeringOrHealth_MastersDegree',
```

Comment:
is there a way we can replace this string with a corrected version, using `DcidFilter.additions`, until this is fixed?